ignition-server/controllers/local_ignitionprovider.go

```diff
@@ -258,18 +258,18 @@ func (p *LocalIgnitionProvider) GetPayload(ctx context.Context, releaseImage str
 			}
 		}
 
-		component = "cluster-config-operator"
-		clusterConfigOperatorImage, ok := imageProvider.ImageExist(component)
+		component = "cluster-config-api"
```

Comment:
This is used as both the image name and binary name, but they are different for the cluster-config-api image.

Reply:
Good spot, thanks. Adressed by ammending the commit.